wgsl/index.bs

```diff
@@ -8556,13 +8556,10 @@ path: syntax/align_attr.syntax.bs.include
     it constrains the byte addresses at which the structure itself and its component members can appear.
 
     <div algorithm="constraint on align attribute">
-    If `align(`|n|`)` is applied to a member of |S|
-    with type |T|,
-    and |S| can be the [=store type=] for variable in address space |AS|,
-    where |AS| is not [=address spaces/uniform=],
+    If `align(`|n|`)` is applied to a member with type |T|,
     then |n| [=shader-creation error|must=] satisfy:
     <blockquote>
-    |n|&nbsp;=&nbsp;|k|&nbsp;&times;&nbsp;[=RequiredAlignOf=](|T|,|AS|)
+    |n|&nbsp;=&nbsp;|k|&nbsp;&times;&nbsp;[=AlignOf=](|T|)
```

Comment:
ok, talked this through with @dj2 

I will revert this change because:
- right now the two phrasings are the same.
- but in future when we want to introduce something like scalar block layout, we need a path to doing that.
    - if we only have Align as a knob, then we have to change Align(vec4u) to be 4.  
    - But then that is also the default alignment used to *set* the layout.  And then that changes the layout of structs that *don't have annotations:  e.g. `{ a:u32, b:vec4u}` would then place b at offset 8 instead of 16.
    -